warehouse/models/mart/gtfs/_mart_gtfs.yml

```diff
@@ -1117,3 +1117,34 @@ models:
         description: '{{ doc("gtfs_stop_times__timepoint") }}'
     meta:
       publish.gis_coordinate_system_epsg: WGS84
+  - name: fct_daily_scheduled_trips
+    description: |
+      A daily table showing all trips that were scheduled on a given date.
+      If a `service_date`, `trip_key` pair is present in this table,
+      it means that that `trip_key` was scheduled to occur on that `service_date`.
+      Specifically, it means that the associated `service_id` was active and had service
+      scheduled. Dates where a `service_id` was active but not scheduled (for example,
+      weekend dates within a weekday service's effective dates) are not listed in this table.
+    columns:
+      - name: key
+        tests:
+          - unique:
+              where: "not warning_duplicate_trip_primary_key"
+          - not_null
```

Comment:
nit: I usually like putting `not_null` first as it's a stronger test than unique; unique excludes nulls

Reply:
I think this is in this order basically everywhere in our entire codebase, I am disinclined to change it here unless you really want me to